networks/suzuka/suzuka-full-node/src/partial.rs

```diff
@@ -196,15 +199,23 @@ where
 	}
 }
 
-async fn read_commitment_events(mut stream: CommitmentEventStream) -> anyhow::Result<()> {
+async fn read_commitment_events<T>(
+	mut stream: CommitmentEventStream,
+	executor: T,
+) -> anyhow::Result<()>
+where
+	T: DynOptFinExecutor + Send + Sync,
+{
 	while let Some(res) = stream.next().await {
 		let event = res?;
 		match event {
 			BlockCommitmentEvent::Accepted(commitment) => {
 				debug!("Commitment accepted: {:?}", commitment);
+				executor.set_finalized_block_height(commitment.height)?;
 			}
 			BlockCommitmentEvent::Rejected { height, reason } => {
 				debug!("Commitment rejected: {:?} {:?}", height, reason);
+				// TODO: block reversion
```

Comment:
More of a note for later but I think we should be adding some more `info!()` traces for logging in production. Here would be a useful place to add one.